core/templates/pages/blog-dashboard-page/modal-templates/blog-card-preview-modal.component.spec.ts

```diff
@@ -96,11 +96,11 @@ describe('Blog Card Preview Modal Component', () => {
   }));
 
   it('should initialize correctly when blog post is not published', () => {
-    sampleBlogPostBackendDict.published_on = null;
+    sampleBlogPostBackendDict.published_on = '';
     blogPostData = BlogPostData.createFromBackendDict(
       sampleBlogPostBackendDict);
     let expectedBlogPostSummary = new BlogPostSummary (
-      blogPostData.id,
+      blogPostData.id as string,
```

Comment:
why is this required? id seems to be a string in `sampleBlogPostBackendDict`

Reply:
It is because of `createInterstitialBlogPost`. But @seanlip advice me not to follow wrong pattern forward. Cover this file later. Hence remove this file from ts file.

Reply:
Correct, but it forced me to use, because throwing error or add guard of (!==) for correct things is not appropriate. (for null case).